test/it/group-get.js

```diff
@@ -2,9 +2,11 @@ const utils = require('../utils');
 const okta = require('../../');
 const expect = require('chai').expect;
 let orgUrl = process.env.OKTA_CLIENT_ORGURL;
+let mockServer = false;
 
 if (process.env.OKTA_USE_MOCK) {
   orgUrl = `${orgUrl}/group-get-and-stats`;
+  mockServer = true;
 }
```

Comment:
Yeah, removed it from each test and moved it to a single check in utils.cleanup() method. Much cleaner now.